libs/components/datetime/src/lib/modules/datepicker/datepicker.component.ts

```diff
@@ -82,6 +82,12 @@ export class SkyDatepickerComponent implements OnDestroy, OnInit {
   public calendarDateRangeChange =
     new EventEmitter<SkyDatepickerCalendarChange>();
 
+  /**
+   * @internal
+   */
+  @Output()
+  public calendarOpenChange = new EventEmitter<boolean>();
```

Comment:
Curious about the solution. Does AG Grid need to stop editing when the field's dropdown closes, or when the field loses focus?

Reply:
AG Grid's feature is called "stop editing when cell loses focus" but our cell uses a custom popup editor in an overlay, outside of AG Grid. Our editor needs to stop editing when neither the cell editor field nor the overlay have focus. In order to track the status of the overlay, I've added these events.

Reply:
Okay, thanks for the explanation.